### DISCUSSION TITLE
Features?

### DISCUSSION THREAD

**Discussion by Jan-517** [marsupialproject/marsupial-emr#2] [General]
I love the idea of a free, open access EMR. Are the "patients" presented in the intro videos part of the system, or is it just empty boxes once downloaded ?  Also, what about security- are there any concerns regarding that ? I would like to be able to share this. Thanks!

**Comment by marsupialproject** [comment]
Either. A blank database is provided, as well as the database in the video for reference.

I'm not sure what you mean by security.  Some of the network security risks are discussed in the Marsupial Instructor video.
